app/doc/DocVersions.scala

```diff
@@ -1,5 +0,0 @@
-package doc
-
-object DocVersions {
-  val Latest = "1.0.x"
```

Comment:
Why do we not encapsulate the versions in an singleton object as before. Currently we are using the versions in multiple places:
- https://github.com/huntc/project-doc/blob/more-versions/app/modules/ConductRDocRendererModule.scala#L36
- https://github.com/huntc/project-doc/blob/more-versions/app/controllers/Application.scala#L90

Therefor we should encapsulate the strings to constants inside of a singleton object.